File: src/com/suushiemaniac/cubing/alglib/transform/rotate/SkewbRotation.java
package com.suushiemaniac.cubing.alglib.transform.rotate;

import com.suushiemaniac.cubing.alglib.move.Move;
import com.suushiemaniac.cubing.alglib.move.SkewbMove;

public enum SkewbRotation implements Rotation {
    //TODO
    ;

    @Override
    public Move rotate(Move origin) {
        return null;
    }

    @Override
    public Move transform(Move origin) {
        return this.rotate(origin);
    }
}

File: src/main/kotlin/com/suushiemaniac/cubing/alglib/transform/rotate/SkewbRotation.kt
package com.suushiemaniac.cubing.alglib.transform.rotate

import com.suushiemaniac.cubing.alglib.move.Move
import com.suushiemaniac.cubing.alglib.move.SkewbMove

enum class SkewbRotation : Rotation {
    ;

    override fun rotate(origin: Move): Move {
        return origin as SkewbMove
    }
}
